### DISCUSSION TITLE
How could I install emacs-head@29 at specific commit rather than head?

### DISCUSSION THREAD

**Discussion by eki3z** [daviderestivo/homebrew-emacs-head#147] [Q&A] [answered]
Same question as https://github.com/d12frosted/homebrew-emacs-plus/issues/441#issuecomment-1067712666
@daviderestivo  How could I local edit emacs-head@29.rb to install at specific commit rather than origin/master ?

**Comment by eki3z** [comment, marked as answer]
Solved. just change revision as ref.

**Reply by daviderestivo** [reply]
@liuyinz would you post the modified code snippet just if someone else is going to ask for the same question. I would assume your code should look like something:

`url "https://brew.sh/foo.git", tag: "1.0", revision: "f5e00e485e7aa4c5baa20355b27e3b84a6912790"`

**Reply by eki3z** [reply]
Oh, just modified the emacs-head@29.rb locally.
delete `revision 1`
change to `url "https://github.com/emacs-mirror/emacs.git", revision: "bc9be5449e1127bc1b05a6cad8471c6eba52c8e9"`
honestly, I don't understand what `revision 1` mean, I just delete it and worked.
